hack/python-sdk/post_gen.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
Is it possible to add guidance or simple README.md to tell how to use this script? 

Honestly, I didn't finish this end to end. I am curious what part is auto generated and which part needs manual work. That would helps other contributions to quickly onboard sdk development

Reply:
Sure, will do that! But I would like to propose we land the updated tooling from #1389 first, then I will update the main `Makefile` with new targets for SDK generation and publishing, and also add a new section to the `README.md` related to Python SDK. WDYT?

Reply:
I think we should execute `post_gen.py` from the [`gen-sdk.sh`](https://github.com/kubeflow/tf-operator/blob/e80031fc3a72fd35aa327724f2303d6b2cd6ae66/hack/python-sdk/gen-sdk.sh#L49) script.
What do you think @Jeffwan @alembiewski ?

Reply:
@andreyvelich, yes, absolutely. The reason I haven't include it into the `gen-sdk.sh` script is that we are waiting for #1389 to land, then I will rebase my PR on top of it and update the tooling.

Reply:
@alembiewski #1389  has been merged